sdk/core/Azure.Core/src/Diagnostics/HttpPipelineEventSource.cs

```diff
@@ -58,7 +58,7 @@ public void Request(Request request)
         [NonEvent]
         public async Task RequestContentAsync(Request request, CancellationToken cancellationToken)
         {
-            if (IsEnabled(EventLevel.Verbose, EventKeywords.None))
+            if (request.Content != null && IsEnabled(EventLevel.Verbose, EventKeywords.None))
```

Comment:
Should we consider guarding against a null request?    `if (request?.Content != null && ...`

Reply:
No, the request is guaranteed to be not-null when we get here.